src/main/target/common_defaults_post.h

```diff
@@ -598,3 +598,11 @@
 #ifndef RTC6705_SPI_INSTANCE
 #define RTC6705_SPI_INSTANCE NULL
 #endif
+
+#ifndef CORE_TEMPERATURE_THRESHOLD
```

Comment:
This should go to where it is actually used.

Reply:
It feels like alarm settings should be a dependency of osd.c/h not part of it?

But sure I can move it to osd.h if you like, next to the other ALARM defines.

Reply:
done in #8490